### PR TITLE
added new graph and format file

### PR DESCRIPTION
**Pull request by doomed-neko**

### CODE REVIEW

**Comment by doomed-neko** [comment]
oh shit that autoformatting messed up shit

**Comment by doomed-neko** [comment]
fixed it now what

File: index.html [hunk 1]
```diff
@@ -1,140 +1,203 @@
-<!DOCTYPE html>
+<!Doctype html>
```
Comment (electron271) [inline index.html:1]: what

File: index.html [hunk 2]
```diff
@@ -161,14 +161,12 @@ <h3 style="text-decoration: underline">COOL LINUX FACT!</h3>
...
+        <p class="graph-description">linux/distro (International system unit)</p>
         <a
           class="graph-download-button"
-          onclick='forceDownload("files/img/arch.png", "arch.png")'
-          ><img src="files/img/aod.png"
+          onclick='forceDownload("files/img/aod.png", "aod.png")'
+          ><img src="files/img/download.png"
         /></a>
       </div>
       <hr />
```
Comment (electron271) [inline index.html:172]: horrifying

**Review by electron271 — CHANGES_REQUESTED** [review]
horrifying formatting, good graph (add watermark if you can though)

File: index.html (hunk 1 above)
Comment (doomed-neko) [reply index.html:1]: idk my autoformatter

**Comment by doomed-neko** [comment]
git is complicated or I might be dumb
I will make a new pr in a minute

**Pull request closed without merging**